patches/net.minecraft.client.settings.GameSettings.java.patch

```diff
@@ -9,7 +67,16 @@
           }
        }
  
-@@ -1025,6 +1025,15 @@
+@@ -335,7 +340,7 @@
+       }
+ 
+       if(settingsOption == GameSettings.Options.PARTICLES) {
+-         this.particleSetting = (this.particleSetting + value) % 3;
++         this.particleSetting = (this.particleSetting + value) % 4;
```

Comment:
doesn't optifine already allow you to disable particles?

Reply:
optifine may not be installed

Reply:
optifine doesnt disable everything

Reply:
well, optifine overrides the particles button, right? or does it just move it to another gui? if it overrides the button, then it's just going to be entirely redundant, i guess i'll check that again once we're done with OF compatibility, but that's one thing im worried about.